src/java.prefs/unix/classes/java/util/prefs/FileSystemPreferences.java

```diff
@@ -596,43 +566,32 @@ private void initCacheIfNecessary() {
      * fails, a BackingStoreException is thrown and both prefsCache and
      * lastSyncTime are unaffected by the call.
      */
-    @SuppressWarnings("removal")
     private void loadCache() throws BackingStoreException {
+        Map<String, String> m = new TreeMap<>();
+        long newLastSyncTime = 0;
         try {
-            AccessController.doPrivileged(
-                new PrivilegedExceptionAction<Void>() {
-                public Void run() throws BackingStoreException {
-                    Map<String, String> m = new TreeMap<>();
-                    long newLastSyncTime = 0;
-                    try {
-                        newLastSyncTime = prefsFile.lastModified();
-                        try (FileInputStream fis = new FileInputStream(prefsFile)) {
-                            XmlSupport.importMap(fis, m);
-                        }
-                    } catch(Exception e) {
-                        if (e instanceof InvalidPreferencesFormatException) {
-                            getLogger().warning("Invalid preferences format in "
-                                                        +  prefsFile.getPath());
-                            prefsFile.renameTo( new File(
-                                                    prefsFile.getParentFile(),
-                                                  "IncorrectFormatPrefs.xml"));
-                            m = new TreeMap<>();
-                        } else if (e instanceof FileNotFoundException) {
-                        getLogger().warning("Prefs file removed in background "
-                                           + prefsFile.getPath());
-                        } else {
-                            throw new BackingStoreException(e);
-                        }
-                    }
-                    // Attempt succeeded; update state
-                    prefsCache = m;
-                    lastSyncTime = newLastSyncTime;
-                    return null;
-                }
-            });
-        } catch (PrivilegedActionException e) {
-            throw (BackingStoreException) e.getException();
+            newLastSyncTime = prefsFile.lastModified();
+            try (FileInputStream fis = new FileInputStream(prefsFile)) {
+                XmlSupport.importMap(fis, m);
+            }
+        } catch(Exception e) {
```

Comment:
```suggestion
        } catch (Exception e) {
```

Reply:
Most catch statements in this file do not have a space.